WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -143,7 +143,7 @@ public void onBindViewHolder(GridViewHolder holder, int position) {
                 if (SiteUtils.isPhotonCapable(mSite)) {
                     thumbUrl = PhotonUtils.getPhotonImageUrl(media.getUrl(), mThumbWidth, mThumbHeight);
                 } else {
-                    thumbUrl = UrlUtils.removeQuery(media.getUrl()) + "?w=" + mThumbWidth + "&h=" + mThumbHeight;
+                    thumbUrl = UrlUtils.removeQuery(media.getUrl());
```

Comment:
> 150px wide pictures are pretty small and look ugly :|

Yep :/

> Maybe do this when on the default device network connection (Not on WiFi, not on any other type of network connection).

That's a good idea. What do you think @bysl?

> We should probably detect when Jetpack is not installed on the self hosted site, and advertise you can save data if you install it, and your site has lot of pictures.

We can detect it, but I'm not sure how to advertise this without being too much "in your face", not doing that in this PR anyway.